events/views.py

```diff
@@ -28,11 +28,24 @@ class HomeView(generic.FormView):
     form_class = forms.SearchHomeForm
     template_name = 'home.html'
 
+    def get_context_data(self, **kwargs):
+        context = super(HomeView, self).get_context_data(**kwargs)
+        if not self.request.user.is_anonymous:
+            profile = models.Profile.objects.get(user_id=self.request.user.id)
+            context['bio'] = profile.bio == '' or profile.bio == None
+            context['first_name'] = self.request.user.first_name == '' or self.request.user.first_name == None
```

Comment:
En Python los strings vacíos son None, así que puedes hacer las comprobaciones directamente con: `not profile.bio`. Por ejemplo.